apps/namadillo/package.json

```diff
@@ -22,6 +22,7 @@
     "framer-motion": "^11.0.28",
     "idb-keyval": "^6.2.1",
     "invariant": "^2.2.4",
+    "io-ts": "^2.2.21",
```

Comment:
@pedrorezende Seems that all of the CI is failing due to this. Maybe we just need to update the `yarn.lock` file?

Reply:
Thanks @jurevans I missed the yarn.lock on my commit!